src/main/java/org/spongepowered/api/item/ItemType.java

```diff
@@ -105,4 +106,6 @@ default ItemStackSnapshot getTemplate() {
         return ItemStack.of(this, 1).createSnapshot();
     }
 
+    Translation getTranslation(ItemStack stack);
```

Comment:
Isn't there already `ItemStack#getTranslation`?

Reply:
Yes. This is used by ItemStack#getTranslation to take advantage of polymorphism since you need the ItemType to translate the ItemStack.

See Item#getTranslationKey(ItemStack stack) in minecraft code for a similar approach.